elements/pfe-datetime/src/pfe-datetime.json

```
@@ -0,0 +1,116 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Datetime",
+  "description": "Datetime is a \"loader\" that indicates to the user that part of the web page is loading, or waiting on other http events to be ready to use.",
+  "type": "object",
+  "tag": "pfe-datetime",
+  "class": "pfe-datetime",
+  "category": "content",
+  "properties": {
+    "slots": {
+      "title": "Slots",
+      "description": "Definition of the supported slots",
+      "type": "object"
+    },
+    "attributes": {
+      "title": "Attributes",
+      "type": "object",
+      "properties": {
+        "datetime": {
+          "title": "Date & Time",
+          "type": "string",
+          "observer": "_basicAttributeChanged"
+        },
+        "type": {
+          "title": "Type",
+          "description": "A formatted time for the indicated locale, or a relative time (1 hour ago)",
+          "type": "string",
+          "enum": [
+            "local",
+            "relative"
+          ]
+        },
+        "timestamp": {
+          "title": "Timestamp",
+          "type": "string"
+        },
+        "weekday": {
+          "title": "Weekday",
+          "type": "string",
+          "enum": [
+            "narrow",
+            "short",
+            "long"
+          ]
+        },
+        "month": {
+          "title": "Month",
+          "type": "string",
+          "enum": [
+            "numeric",
+            "2-digit",
+            "narrow",
+            "short",
+            "long"
+          ]
+        },
+        "day": {
+          "title": "Day",
+          "type": "string",
+          "enum": [
+            "numeric",
+            "2-digit"
+          ]
+        },
+        "year": {
+          "title": "Year",
+          "type": "string",
+          "enum": [
+            "numeric",
+            "2-digit"
+          ]
+        },
+        "hour": {
+          "title": "Hour",
+          "type": "string",
+          "enum": [
+            "numeric",
+            "2-digit"
+          ]
+        },
+        "minute": {
+          "title": "Minute",
+          "type": "string",
+          "enum": [
+            "numeric",
+            "2-digit"
+          ]
+        },
+        "second": {
+          "title": "Second",
+          "type": "string",
+          "enum": [
+            "numeric",
+            "2-digit"
+          ]
+        },
+        "locale": {
+          "title": "Locale",
+          "description": "A locale code like en-US or en-GB",
+          "type": "string",
+          "default": "en-US"
+        },
+        "timeZoneName": {
```

Schemas use underscores for variable keys rather than camel case: https://json-schema.org/understanding-json-schema/reference/object.html.
```suggestion
        "timezone_name": {
```

---

@castastrophe This is done.